conans/paths/__init__.py

```diff
@@ -12,7 +12,21 @@
 
 
 def get_conan_user_home():
-    user_home = os.getenv("CONAN_HOME")
+    def _read_user_home_from_rc():
+        try:
+            conanrc_path = os.path.join(os.getcwd(), "conan.conanrc")
```

Comment:
I think the file can be called just ``.conanrc``
```suggestion
            conanrc_path = os.path.join(os.getcwd(), ".conanrc")
```